packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/virtualMachine/dedicatedCloud-datacenter-virtualMachine.service.js

```diff
@@ -0,0 +1,15 @@
+export default class {
+  /* @ngInject */
+  constructor(OvhHttp) {
+    this.OvhHttp = OvhHttp;
+  }
+
+  deleteLicense(serviceName, datacenterId, vmId) {
+    return this.OvhHttp.post(
+      `/dedicatedCloud/${serviceName}/datacenter/${datacenterId}/vm/${vmId}/removeLicense`,
+      {
+        rootPath: 'apiv6',
+      },
+    );
```

Comment:
Why post and not delete http verb ?

```suggestion
    return this.$http.post(
      `/dedicatedCloud/${serviceName}/datacenter/${datacenterId}/vm/${vmId}/removeLicense`);
```